airflow/sensors/hdfs_sensor.py

```diff
@@ -45,6 +51,13 @@ def __init__(self,
                  *args,
                  **kwargs):
         super().__init__(*args, **kwargs)
+        if not snakebite_loaded:
+            raise ImportError(
+                'This HdfsSensor implementation requires snakebite, but '
+                'snakebite is not compatible with Python 3 '
+                '(as of August 2015). Please use Python 2 if you require '
+                'this hook  -- or help by submitting a PR!')
```

Comment:
General question: I always wonder why there's code this short (75 chars) throughout the code base while the configured line length in Airflow is 110 characters. Is it enforced by some IDE default?

Reply:
I think @mik-laj uses vim mostly ;). But indeed if it was IntelliJ I had to set line length to 110 in my IDE from the default 80 so I guess that's the reason.

Reply:
This is a copy-paste from HDFS Hook:
https://github.com/apache/airflow/blob/master/airflow/hooks/hdfs_hook.py#L48-L52
I will correct it to take up to 110 characters.